### ISSUE TITLE
You didn't finish it!

### ISSUE THREAD

**Issue by Gibbonator** [devonwhale/advent-of-code-2022#1]

**Comment by devonwhale** [comment]
You are right. Had a bit of stuff come up that needed addressing come up and then it didn't feel right to skip ahead but I didn't have time to go back and do the old ones. It's on the list though, so hopefully one day...

**Comment by Gibbonator** [comment]
We will watch your career with great interest
